src/preferences/dialog/dlgprefwaveformdlg.ui

```diff
@@ -490,6 +491,42 @@ Select from different types of displays for the waveform, which differ primarily
        </property>
       </widget>
      </item>
+     <item row="5" column="0">
+        <widget class="QLabel" name="label">
+        <property name="text">
+                <string>Play Marker Position</string>
+        </property>
+        </widget>
+        </item>
+        <item row="5" column="1" colspan="3">
+        <widget class="QSlider" name="playMarkerPositionSlider">
```

Comment:
Since the slider is not labeled, what about a small tooltip? We provide them in the preferences for at least some of the more recently added controls.  Similar to ``"Moves the playposition marker on the waveforms to the left, 
 right or center (default).``. It gets better but you get the idea :)

Reply:
all done.